app/src/index.css

```diff
@@ -56,7 +56,7 @@
 
 /* Add styles to components */
 @layer components {
-  .indent {
+  .intent {
```

Comment:
this seems like a really generic name ! 
could we do a different name ?

Reply:
i think i mixed up also   .intent with .indent ...  so "indent" or "indentation" is the right world in english.
but as we wana use this just for <p> with long TEXT  i renamed it 

.text-indent